posts/project-mycorrhiza/index.md

```diff
@@ -0,0 +1,188 @@
+authors:
+  - venefilyn
+  - flachman
+tags:
+  - Project Mycorrhiza
+  - dashboard
+  - ux
+  - testing-farm
+  - logs
+  - copr
+---
+
+This term has been thrown around a bit in issues here and there relating to the dashboard and
+people usually get confused, and rightfully so! Lets dive in and see what Project Myccorhiza
+is and why we chose this name.
+
+![Photo of beech trees in a forest](./img/forest.jpg "New beech leaves, Gribskov Forest in the northern part of Sealand, Denmark.")
+Photo by [Malene Thyssen](https://commons.wikimedia.org/wiki/User:Malene) - Own work, [CC BY-SA 3.0](http://creativecommons.org/licenses/by-sa/3.0/ "Creative Commons Attribution-Share Alike 3.0"), [Link](https://commons.wikimedia.org/w/index.php?curid=752225)
+
+<!--truncate-->
+
+---
+
+[TL;DR at the bottom!](#tldr)
+
+Before I explain what Project Mycorrhiza is and what it's about, lets talk about
+the Packit dashboard and why it's been getting so much love from me the past year.
+
+It's to no surprise that Packit is used within a lot of different projects around
+Fedora, which includes [Convert2RHEL](https://github.com/oamg/convert2rhel) that I'm
+a developer of. But I've always felt like Packit's dashboard has missed the mark on
+being the go-to for getting an overview for the pipeline, results, or even the
+project itself.
+
+Since I like making websites better for my own benefit, I've spending a huge amount of my own
+time the past year to gradually improve the Packit dashboard to prepare it for something
+bigger. Which is where Project Mycorrhiza comes in.
+
+## What's mycorrhiza and mycorrhizal network?
+
+:::note
+
+This is heavily simplified for the purpose of getting a quick overview. I encourage you
+to read more about [mycorrhiza on the wikipedia article about it](https://en.wikipedia.org/wiki/Mycorrhiza)
+and any related articles!
+
+:::
+
+You might be familiar with something called mycelium, the network of root-like structures that
+fungi form. I'm assuming most of you have seen this form on food before, such as bread
+that grew mold.
+
+The mycelia helps fungi get nutrients and get enough food to grow mushrooms!
+Some are just terrible to be around, some are interesting to look at but relatively toxic, and
+some are even edible. Those same labels I applied to the mushrooms are something plants and trees
+sort of do as well!
+
+A mycorrhizal network is a symbiosis between fungi and flowers and trees.
+You have already seen one too as the photo above is a bunch of Beech trees that
+are "[ectomyccorhizal](https://en.wikipedia.org/wiki/Ectomycorrhiza)"!
+What it means is that nutriets are shared between the two, the fungi spreads out mycelia in the soil
+and transfers nutrients to the tree, and the tree provides sugar and fat back to the fungi.
+
+Since my project relates to integrating more services and getting a better
+packaging ecosystem, I chose the name Project Mycorrhiza to put emphasis on symbiosis between
+the different tools.
```

Comment:
```suggestion
Since my project relates to integrating more services and getting a better
packaging ecosystem, I chose the name Project Mycorrhiza to put emphasis on the symbiosis between
the different tools.
```